sdks/python/setup.py

```diff
@@ -136,11 +136,17 @@ def get_version():
     'pyhamcrest>=1.9,<2.0',
     ]
 
+GCP_REQUIREMENTS_PY2_ONLY = [
+    'googledatastore==7.0.1'
+]
+
+GCP_REQUIREMENTS_PY3_ONLY = [
+]
+
 GCP_REQUIREMENTS = [
     # oauth2client >=4 only works with google-apitools>=0.5.18.
     'google-apitools>=0.5.18,<=0.5.20',
     'proto-google-cloud-datastore-v1>=0.90.0,<=0.90.4',
-    'googledatastore==7.0.1',
```

Comment:
We recently learned (thanks to @manuzhang ) that we don't need to split the requirements in multiple arrays - please add a version qualifier same as in https://github.com/apache/beam/pull/6543.

Reply:
I have updated my setup.py to the latest version in the master for testing purposes and have added a version qualifier instead. Thanks for the feedback. PTAL at the next commit

Reply:
Is there still an issue with py27-gcp?